src/System.Text.Encoding.CodePages/src/System/Text/SBCSCodePageEncoding.cs

```diff
@@ -37,6 +37,27 @@ public SBCSCodePageEncoding(int codePage, int dataCodePage) : base(codePage, dat
         {
         }
 
+        private static unsafe void ZeroMemAligned(byte* buffer, int count)
+        {
+            long* pLong = (long*)buffer;
+            long* pLongEnd = (long*)(buffer + count - sizeof(long));
+
+            while (pLong < pLongEnd)
+            {
+                *pLong = 0;
+                pLong++;
+            }
+
+            byte* pByte = (byte*)pLong;
+            byte* pEnd = buffer + count;
+
+            while (pByte < pEnd)
+            {
+                *pByte = 0;
+                pByte++;
+            }
+        }
+
```

Comment:
Talk to Marcos or Ulises who may help with running the test on ARM. if it is not easy, you may integrate the change to PN and watch the test results there.